### DISCUSSION TITLE
Set Alerts not filtered by message level ?

### DISCUSSION THREAD

**Discussion by clabnet** [datalust/seq-tickets#1473] [Help]
Hi, 
I have tried to set an alert with a signal based on AppName AND level message, but it seems that is triggered for ALL messages with selected AppName. I missing other things ? 
My goal is to trigger the alaert only for Warning messages. 
Thanks

The signal for AppName was named Prova Tutti

![signal-provatutti](https://user-images.githubusercontent.com/6067610/152549911-bc3de0e2-f712-47f6-8ffd-b67e501f8c0f.png)

Applied on it the level filter Warnings

![signal-provatutti-warnings](https://user-images.githubusercontent.com/6067610/152550085-d902db7b-1a40-4983-a69a-d67897d17afe.png)
 
 The alert is this 
 
![alert](https://user-images.githubusercontent.com/6067610/152550273-b6a6ee3e-4f43-4f78-9bfe-979c7b2f1692.png)

**Comment by nblumhardt** [comment]
Thanks for getting in touch, @clabnet. Your alert configuration looks correct to me; if you click the little "play" button next to the alert editor, do you see the results that you expect?

I can't reproduce the problem locally, so it might help to share some more details of the underlying data and the alert notifications so we can narrow down possible causes (feel free to email `support@` if you would rather not post more details here).

Best regards,
Nick
